app/bot.py

```diff
@@ -4,58 +4,162 @@
 from discord.ext import commands
 from dotenv import load_dotenv
 
-bot = commands.Bot(command_prefix='/')
 
-def main():
-    load_dotenv()
-    sys.stdout.flush()
-    bot.run(os.getenv("TOKEN"))
+reaction_emoji = '📌'
+
+prefix_map = {
+    reaction_emoji: ''
+}
```

Comment:
I was aiming to allow this to support the other pin-like emojis, but after thinking about it, I think that’s  might make things a lot more complicated, seeing as how you'd have to keep track or different reaction stacks. However, it was designed with that in mind.